theme_kiddo/views/snippets/s_cover.xml

```diff
@@ -4,7 +4,7 @@
 <template id="s_cover" inherit_id="website.s_cover">
     <!-- Section -->
     <xpath expr="//section" position="attributes">
-        <attribute name="class" add="o_full_screen_height o_cc o_cc1" remove="bg-black-50" separator=" "/>
+        <attribute name="class" add="o_full_screen_height o_cc1" remove="bg-black-50 o_cc4" separator=" "/>
```

Comment:
Buttons are not very visible

| this pr | master |
|--------|--------|
| ![image](https://github.com/user-attachments/assets/71d2be73-3c06-4bdf-ba84-02a033b7b8ea) | ![image](https://github.com/user-attachments/assets/70b539c1-3dd6-40ea-bbda-ea86996103fb) |